astropy/coordinates/representation/base.py

```diff
@@ -426,6 +451,45 @@ def shape(self, shape):
                 else:
                     reshaped.append(val)
 
+    @property
+    def masked(self):
+        return isinstance(getattr(self, self.components[0]), Masked)
+
+    def _ensure_masked(self):
+        """Ensure Masked components."""
+        # TODO: should we just allow the above property to be set?
+        if not self.masked:
+            for comp in self.components:
+                c = "_" + comp
+                setattr(self, c, Masked(getattr(self, c)))
+
+    def get_mask(self, *attrs):
+        """Calculate the mask, by combining masks from the given attributes.
+
+        Parameters
+        ----------
+        *attrs : str
+            Attributes from which to get the masks to combine. If not given,
+            use all components of the class.
+
+        Returns
+        -------
+        mask : ~numpy.ndarray of bool
+            The combined, read-only mask. If the instance is not masked, it
+            is an array of `False` with the correct shape.
+        """
+        if not attrs:
+            attrs = self.components
```

Comment:
```suggestion
        attrs = self.components if not attrs else attrs
```

Reply:
I find that less clear. Wish the `?=` operator had succeeded...